test/boxes.json

```diff
@@ -1,17 +0,0 @@
-[
```

Comment:
🗒️ This JSON file defined 3 different host VMs. However, in practice, only 1 of these was ever used in the functional tests. This was extra complexity wasn't serving a purpose, so I removed the JSON in favor of just hardcoding a single host.